Mathlib/Geometry/RingedSpace/LocallyRingedSpace.lean

```diff
@@ -74,37 +74,26 @@ structure Hom (X Y : LocallyRingedSpace.{u}) : Type _ where
   /-- the underlying morphism between ringed spaces -/
   val : X.toSheafedSpace ⟶ Y.toSheafedSpace
   /-- the underlying morphism induces a local ring homomorphism on stalks -/
-  prop : ∀ x, IsLocalRingHom (PresheafedSpace.stalkMap val x)
+  prop : ∀ x, IsLocalRingHom (val.stalkMap x)
 
 instance : Quiver LocallyRingedSpace :=
   ⟨Hom⟩
 
 @[ext] lemma Hom.ext' (X Y : LocallyRingedSpace.{u}) {f g : X ⟶ Y} (h : f.val = g.val) : f = g :=
   Hom.ext _ _ h
 
--- TODO perhaps we should make a bundled `LocalRing` and return one here?
--- TODO define `sheaf.stalk` so we can write `X.𝒪.stalk` here?
-/-- The stalk of a locally ringed space, just as a `CommRing`.
--/
-noncomputable def stalk (X : LocallyRingedSpace.{u}) (x : X) : CommRingCat :=
-  X.presheaf.stalk x
-
--- Porting note (#10754): added this instance to help Lean realize stalks are local
--- (so that `0 ≠ 1` works below)
-instance stalkLocal (x : X) : LocalRing <| X.stalk x := X.localRing x
-
 /-- A morphism of locally ringed spaces `f : X ⟶ Y` induces
 a local ring homomorphism from `Y.stalk (f x)` to `X.stalk x` for any `x : X`.
 -/
-noncomputable def stalkMap {X Y : LocallyRingedSpace.{u}} (f : X ⟶ Y) (x : X) :
-    Y.stalk (f.1.1 x) ⟶ X.stalk x :=
-  PresheafedSpace.stalkMap f.1 x
+noncomputable def Hom.stalkMap {X Y : LocallyRingedSpace.{u}} (f : Hom X Y) (x : X) :
```

Comment:
I am not a big fan of the `Hom` hack here. I think this should be brought up with core but shouldn't hold the PR back.

Reply:
What do you mean by the `Hom` hack? Do you mean using `(f : Hom X Y)` instead of `(f : X ⟶ Y)` to make dot notation work?

Reply:
Yes

Reply:
I'm not that happy about it too. "`Quiver.Hom` is a semireducible def so it is unreasonable to tell the dot notation to see through it". This is the answer I got the last time I asked.